package.json

```diff
@@ -19,7 +19,7 @@
   "homepage": "https://frappeframework.com",
   "dependencies": {
     "ace-builds": "^1.4.8",
-    "air-datepicker": "http://github.com/frappe/air-datepicker",
+    "air-datepicker": "git+https://github.com/frappe/air-datepicker.git",
```

Comment:
Seems like you can just do the following for [Github URLs](https://docs.npmjs.com/files/package.json#github-urls) (as long as you have npm 1.1.65+):

```suggestion
    "air-datepicker": "frappe/air-datepicker",
```

Reply:
`npm install frappe/air-datepicker` works as well. Only `npm install http://github.com/frappe/air-datepicker` fails with the permission denied error.

I'd prefer `github:frappe/air-datepicker` in that case because its shows where the dependency actually comes from (gitlab, github, bitbucket, ...) and what this is about.